src/configs/redoc/customer-accounts-api/spec.yml

```diff
@@ -0,0 +1,249 @@
+                  Path=/customer; Domain=accounts.livechat.com; Expires=Sun, 18 Jun 2023 12:26:02 GMT;
+                  Max-Age=63072000; HttpOnly; Secure; SameSite=None
+          content:
+            application/json:
+              schema:
+                description: OAuth 2 `Bearer` token
+                type: object
+                properties:
+                  access_token:
+                    type: string
+                    description: Access token value
+                    example: 'fra-a:vky95357S7KM1E4dXwv1DQ'
+                  client_id:
+                    type: string
+                    format: hex
+                    description: Client ID for which the token was issued
+                    example: c6e4f62e2a2dab12531235b12c5a2a6b
+                  entity_id:
+                    type: string
+                    format: uuid
+                    description: Entity ID for which the token was issued
+                    example: 3aa138c1-c137-41c6-6b26-cface5857378
+                  expires_in:
+                    type: integer
+                    format: uint64
+                    description: Access token lifetime (in seconds)
+                    example: 28800
+                  token_type:
+                    type: string
+                    description: Access token type
+                    example: Bearer
+        '401':
+          description: Unauthorized, missing or invalid authorization
+          content:
+            application/json:
+              schema: *ref_0
+        '500':
+          description: Server Error
+    delete:
+      summary: Revoke token
+      description: >-
+        Revoke a token provided via authorization
+        The access token can be provided via authorization.
+      security:
+        - OAuth2 Bearer Token: []
+      tags:
+        - Tokens
+      responses:
+        '200':
+          description: 'OK: Returns no content.'
+        '401':
+          description: Unauthorized, missing or invalid authorization
+          content:
+            application/json:
+              schema: *ref_0
+        '500':
+          description: Server Error
+components:
+  schemas: {}
+  securitySchemes:
+    OAuth2 Bearer Token:
+      description: This API uses OAuth2 token for customers
+      type: oauth2
+      flows:
+        implicit:
+          authorizationUrl: 'https://accounts.livechat.com/customer'
+    OAuth2 Agent Bearer Token:
+      description: >-
+        This API uses OAuth2 for agents with the implicit grant flow. <a
+        href="https://developers.livechat.com/docs/authorization/authorizing-api-calls/#implicit-grant">Learn
+        about the implicit grant flow.</a>
+      type: oauth2
+      flows:
+        implicit:
+          authorizationUrl: 'https://accounts.livechat.com'
+        authorizationCode:
+          authorizationUrl: 'https://accounts.livechat.com'
+          tokenUrl: 'https://accounts.livechat.com/token'
+    Cookie Auth:
```

Comment:
```suggestion
    Cookie Grant:
```

Reply:
I have changed the order but I prefer "Auth" here, this will still be true, take a look please: https://swagger.io/docs/specification/authentication/cookie-authentication/

Reply:
So maybe we should name our this flow in our docs "Cookie Auth"? Or "Cookie Auth Grant"?